README.md

```diff
@@ -218,6 +226,124 @@ func userNotificationCenter(
 
 If your user taps on the notification this will be tracked back to Klaviyo as an "Opened Push" event assuming you have implemented the tracking changes discussed above.
 
+## Handling deep linking
+
+There are two use cases for deep linking that can be relevant here - 
+1. When you push a notification to your app with a deep link.
+2. Any other cases where you may want to deep link into your app via SMS, email, web browser etc.
+
+Note that Klaviyo doesn't officially yet support universal links but since there is no validation on the klaviyo front end for URI schemes, one can include universal links but making sure it works as per expectations is on the developer.
+
+In order for deep linking to work, there are a few configurations that are needed and these are no different from what are required for handling deep linking in general and [Apple documentation](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) on this can be followed in conjunction with the steps highlighted here - 
+
+### Step 1: Register the URL scheme
+
+In order for Apple to route a deep link to your application you need to register a URL scheme in your application's info.plist file. This can be done using the editor that xcode provides in xcode from the Info tab of your project settings (reference screenshot below) or editing the info.plist directly - 
+
+![](example1.png)
```

Comment:
I'm wary of putting screenshots in. They become stale very quickly.

Reply:
Yeah I feel the same but I felt it's easier to understand with the screenshot as to where to find the URL scheme. May be I'm overdoing this readme and this is probably support to be part of the support article. Do you know if we have one on deep linking?

Reply:
We don't, but I think we could live without screenshots until we have that support article in place. Shouldnt be too hard for an iOS developer to understand where their Info.plist lives I hope

Reply:
So that's the thing, you could directly add this to the `Info.plist` but xcode provides you a UI under the app's main target where you can go add this in and then that translates to XML in the Info.plist. This is what [apple's official documentation](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app#Register-your-URL-scheme) on this refers to as well. While this too might be straightforward I'm often quite lost in the xcode's application level settings since there are so many and it's just not very easy to find stuff.

All that said, I do agree that screenshots are hard to maintain. Would linking to Apple's official docs on this make more sense?

Reply:
That makes sense to me anyway!!

Reply:
I've actually linked apple's documentation on this already. I just noticed that. I'll go ahead and remove the screenshots.